tests/roxctl/bats-tests/helpers.bash

```diff
@@ -226,10 +227,13 @@ image_reference_regex() {
   local component="$2"
   local version="${3:-$any_version}"
 
-  case $registry_slug in
-    quay.io)
+  case "$registry_slug" in
+    quay.io/rhacs-eng)
```

Comment:
I think, there may be still some test cases that would use the old slug `quay.io`. We need to either change the assertions to assert against `quay.io/rhacs-eng` or  change this line to handle both cases: 
```suggestion
    quay.io|quay.io/rhacs-eng)
```